forumone/wp-cfm
# chore(linting): Adds Linting Support & Sets a Baseline

### All Submissions:

* [x] Have you followed the [plugin Contributing guideline](https://github.com/forumone/wp-cfm/blob/develop/CONTRIBUTING.md)?





### Changes proposed in this Pull Request:



* Adds PHPStan & PHP_Code_Sniffer linting support.
* Generates a PHPStan baseline to ensure code quality going forward.
* Applies automatic WordPress Coding Standards automatic fixes.
* Adds initial scaffolding to begin implementing unit testing.
* Fixes GitHub pull requests template.

### How to test the changes in this Pull Request:

1. Ensure Node v14, PHP 8.0, & Composer are installed on your machine.
2. Run `composer install` & `npm ci`
3. Run `composer lint` & `composer analyze`.
4. You should get linting results.

### Other information:

* [N/A] Have you written new tests for your changes, as applicable?



### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
